src/main/utils/mavenUtils.ts

```diff
@@ -106,13 +106,10 @@ export class MavenUtils {
 
     /**
      * Get an array of [groupId, artifactId, version] from dependencies tree node.
-     * @param dependenciesTreeNode - The dependencies tree node
+     * @param dependency - The dependencies tree node
      */
-    public static getGavArray(dependenciesTreeNode: DependenciesTreeNode): string[] {
-        return dependenciesTreeNode.generalInfo
-            .getComponentId()
-            .toLowerCase()
-            .split(':');
+    public static getGavArray(dependency: DependencyIssuesTreeNode): string[] {
```

Comment:
```suggestion
    public static getGavArray(dependencyTreeNode: DependencyIssuesTreeNode): string[] {
```

Reply:
I don't think we should repeat ourselves as the type can tell us it is a tree node.